eip.tf

```diff
@@ -1,12 +1,8 @@
 #Elastic IP nat
 resource aws_eip "nat" {
-count = length(var.subnets_cidr_private)
-vpc = true
-}
+  count = length(var.subnets_cidr_private)
 
-#Elastic IP Public 
-  resource "aws_eip" "pub_ip" {
-  count = 2
-  instance = aws_instance.reut_web[count.index]
-  vpc      = true
```

Comment:
I deleted all that resource. It is not used and is not needed.

Reply:
because I can just declare it in the web instances ? bacause I need them to have a public IP address.

Reply:
So just put associate public ip as true as you did .